ui/app/styles/components/namespace-picker.scss

```diff
@@ -5,65 +5,33 @@
 
 .namespace-picker {
   position: relative;
-  color: $white;
+  color: var(--token-color-palette-neutral-300);
   display: flex;
-  fill: $white;
   padding: $spacing-xxs $spacing-xs;
   width: 100%;
-
-  @include from($mobile) {
-    margin-left: -$spacing-xs;
-    padding: $spacing-xxs 0 $spacing-xxs $spacing-s;
-    width: auto;
-  }
```

Comment:
Since the sidebar collapses on smaller breakpoints we don't need the namespace picker to be responsive

Reply:
I love to hear this!